db.php

```diff
@@ -15,7 +15,7 @@
 require_once $configFile;
 
 // Create connection
-$conn = new mysqli($servername, $username, $password, $dbname);
```

Comment:
here no need to change value. 
Copy config.sample.php to config.php (cp config.sample.php config.php)
Set up your own database credentials in config.php

Reply:
Hey! Can you check now?